helpers.lua

```diff
@@ -24,6 +24,26 @@ end
 
 local _G = ShaguTweaks.GetGlobalEnv()
 
+--[[
+    local lang = GetLocale()
+    ShaguTweaks.T = setmetatable(ShaguTweaks_translation[lang] or {}, { __index = function(tab,key)
+      local value = tostring(key)
+      rawset(tab,key,value)
+      return value
+    end})
+--]]
+local lang = GetLocale()
+local locale = ShaguTweaks_locale[lang] or {}
+local translation = ShaguTweaks_translation[lang] or {}
+
+ShaguTweaks.L = setmetatable(locale, { __index = function(tab,key)
+  local value = tostring(key)
+  rawset(tab, key, value)
+  return value
+end})
+
+ShaguTweaks.T = setmetatable(translation, { __index = ShaguTweaks.L })
+
```

Comment:
Putting that meta table on the `locales` could break things. I think to put that into the main.lua and only on the translation table would be better:

```diff
diff --git a/main.lua b/main.lua
index 1d5c4ff..bfa4d86 100644
--- a/main.lua
+++ b/main.lua
@@ -18,6 +18,15 @@ ShaguTweaks.mods = {}
 
 -- load translation tables
 ShaguTweaks.L = (ShaguTweaks_locale[GetLocale()] or ShaguTweaks_locale["enUS"])
+ShaguTweaks.T = (ShaguTweaks_translation[GetLocale()] or ShaguTweaks_translation["enUS"])
+
+-- use table index key as translation fallback
+ShaguTweaks.T = setmetatable(ShaguTweaks.T, { __index = function(tab,key)
+  local value = tostring(key)
+  rawset(tab, key, value)
+  return value
+end})
+
 ShaguTweaks:RegisterEvent("VARIABLES_LOADED")
 ShaguTweaks:SetScript("OnEvent", function()
   -- load current expansion
```

As a fictional example, checking if the nameplate belongs to a critter and should be hidden, one could do
if L["critters"][nameplate:GetName()] then nameplate:Hide() end

If that metatable would be set on the `L` it would always return true, as it finds **everything** as it always returns the index when there is nothing. Checks like this happen quite often on those tables, so I'd like to not have the metatable on it :)

Reply:
Will you make changes? or should I make a commit? I don't know how Review mode works.

Reply:
> I don't know how Review mode works.

Me neither, I didn't meant to start a review mode, just wanted to comment on where i spotted it.
You do the commit please.